<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_tank_order</class>
 <widget class="QWidget" name="widget_tank_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The order of the tanks in the table determines how eager the auto-ballast algorithm is to fill that tank.&lt;/p&gt;&lt;p&gt;Tanks higher on the list are more likely to be filled than tanks lower on the list.&lt;/p&gt;&lt;p&gt;The order of the list can be changed by using one of the buttons or by dragging the tanks up or down in the list by selecting their name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbElevation">
     <property name="text">
      <string>Order by elevation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbMinimizeRadii">
     <property name="text">
      <string>Minimize radii of gyration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbMaximizeRadii">
     <property name="text">
      <string>Maximize radii of gyration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dsX">
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dsX_2">
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dsX_3">
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbNearest">
     <property name="text">
      <string>Nearest to point</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbFurthest">
     <property name="text">
      <string>Furthest from point</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbElevation</tabstop>
  <tabstop>pbMinimizeRadii</tabstop>
  <tabstop>pbMaximizeRadii</tabstop>
  <tabstop>dsX</tabstop>
  <tabstop>dsX_2</tabstop>
  <tabstop>dsX_3</tabstop>
  <tabstop>pbNearest</tabstop>
  <tabstop>pbFurthest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
